tools/build/pack_windows.py

```diff
@@ -392,35 +392,39 @@ def packMsi(build_json=None, app_src=None, app_dest=None, app_name=None):
 
     os.chdir(BUILD_ROOT)
     pkg_name = "org.xwalk." + app_name.replace("-", "")
-    project_root = os.path.join(BUILD_ROOT, pkg_name)
+    #project_root = os.path.join(BUILD_ROOT, pkg_name)
     crosswalk_app_tools = os.getenv("CROSSWALK_APP_TOOLS")
     if crosswalk_app_tools == None:
         LOG.error("Pls add an environment variable named 'CROSSWALK_APP_TOOLS', and set the crosswalk-app-tools path to this environment variable")
         os.chdir(orig_dir)
         return False
-    create_cmd = "node %s/src/crosswalk-app create %s --platforms=windows --windows-crosswalk=crosswalk-%s.zip" \
-	        % (crosswalk_app_tools, pkg_name, CROSSWALK_VERSION)
 
-    if not doCMD(create_cmd, DEFAULT_CMD_TIMEOUT):
-        os.chdir(orig_dir)
-        return False
+    if not safelyGetValue(build_json, "apk-type") or safelyGetValue(build_json, "apk-type") != "MANIFEST":
+        if os.path.exists(os.path.join(app_src, "manifest.json")):
+            if not doRemove([os.path.join(app_src, "manifest.json")]):
+                os.chdir(orig_dir)
+                return False
+        if os.path.exists(os.path.join(app_src, "icon.ico")):
+            build_cmd = "node %s/src/crosswalk-pkg -c crosswalk-%s.zip --platforms=windows -m " \

```

Comment:
-c parameter, we could use CROSSWALK_VERSION to download crosswalk-*.zip. by itself